packages/cfpb-forms/src/organisms/multiselect.less

```diff
@@ -99,6 +99,7 @@ select.o-multiselect {
   }
 
   &_options {
+    list-style-type: none;
```

Comment:
Unlike https://github.com/cfpb/consumerfinance.gov/pull/7205, I put this on the `ul` at all times, just in case we ever have other messages in the future. If this seems like a bad idea, we can go with 7205.